config/redis.yml

```diff
@@ -11,9 +11,6 @@ development: &defaults
   sm_store:
     namespace: sm-service
     each_ttl: 1200
-  user_store: # THIS WILL BE RETIRED
-    namespace: users
-    each_ttl: 3600
   user_b_store:
     namespace: users_b
```

Comment:
Is there a way to get this back to being 'users'? Obviously low priority, more for future developers "why is this users_b"? 😉

Reply:
hahaha, yea, there is but if we would like to do that, then I'd have to change OldUser to UpdatedUser, have it inherit from user, override the user store declaration, have it persist for X number of hours. Then change get rid of UpdatedUser and have User point to the new user store. Kind of a mess.